### DISCUSSION TITLE
Strange results with the newest NVidia driver (v531)

### DISCUSSION THREAD

**Discussion by sapo17** [mitsuba-renderer/mitsuba3#647] [General]
### System information:

  OS: Windows-10
  CPU: Intel64 Family 6 Model 158 Stepping 10, GenuineIntel
  GPU: NVIDIA GeForce RTX 2060
  Python: 3.9.13 (tags/v3.9.13:6de2ca5, May 17 2022, 16:36:42) [MSC v.1929 64 bit (AMD64)]
  NVidia driver: 528.49
  CUDA: 11.8.89
  LLVM: 15.-1.-1

  Dr.Jit: 0.4.1
  Mitsuba: 3.2.1
     Is custom build? False
     Compiled with: MSVC 19.34.31942.0
     Variants:
        scalar_rgb
        scalar_spectral
        cuda_ad_rgb
        llvm_ad_rgb

### Description

Hello, so this is not really a question, but rather something that I encountered, and I thought of sharing it.

The other day I upgraded my NVidia driver to the newest version (531.31). After that, some of the material optimization test cases that I had written for my thesis failed (using the cuda variant). I will go not into detail about them, but please let me know if you need further information.

If someone else has a familiar problem, changing the NVidia version to a previous version might help. In my case, I downgraded to 528.49, and now my test cases work as previously.

**Comment by njroussel** [comment]
Hi @sapo17 

What do you mean by your optimizations "failed"? Did they crash or produce wrong results?

**Reply by sapo17** [reply]
Hello, 
the optimization procedure looks like [this](https://mitsuba.readthedocs.io/en/stable/src/inverse_rendering/gradient_based_opt.html), but my test cases were different (using adifferent scenes and material parameters).
They produced **wrong results**. Although once in a while I experience a similar error like this #587 (but this is an independent topic).

**Reply by njroussel** [reply]
Ok, thank you.

Unfortunately, v531 hasn't been released on Linux yet - I cannot test this for myself. I will keep this issue open.
